packages/chakra-ui/src/ErrorList/ErrorList.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { ErrorListProps } from "@rjsf/core";
+import {
+  Heading,
+  List,
+  ListIcon,
+  ListItem,
+  Alert,
+  AlertTitle,
+} from "@chakra-ui/react";
+import { WarningIcon } from "@chakra-ui/icons";
+
+const ErrorList = ({ errors }: ErrorListProps) => {
```

Comment:
To be consistent with other themes -- let's decrease the size of the "Errors" title so it is smaller than the actual form title.

![image](https://user-images.githubusercontent.com/1689183/152564407-7a055d06-d10d-4057-b930-aaa95697e168.png)

Reply:
I updated it to:

![image](https://user-images.githubusercontent.com/1689183/152565451-c772c373-5e70-4436-92ce-961d1e949fb7.png)